pkg/services/live/live.go

```diff
@@ -199,26 +194,8 @@ func (g *GrafanaLive) Init() error {
 			return
 		}
 
-		dto := models.UserProfileDTO{
-			Id:             user.UserId,
-			Name:           user.Name,
-			Email:          user.Email,
-			Login:          user.Login,
-			IsGrafanaAdmin: user.IsGrafanaAdmin,
-			OrgId:          user.OrgId,
-		}
-
-		jsonData, err := json.Marshal(dto)
-		if err != nil {
-			logger.Debug("error reading user", "dto", dto)
-			ctx.Resp.WriteHeader(404)
-			return
-		}
-		logger.Info("Logged in user", "user", user)
-
 		cred := &centrifuge.Credentials{
 			UserID: fmt.Sprintf("%d", user.UserId),
-			Info:   jsonData,
```

Comment:
I am removing this info for now -- it is not used and adds extra bytes to messages that do not need it!